landscaper/pkg/controlplane/apis/imports/validation/validation_test.go

```diff
@@ -66,34 +76,79 @@ var _ = Describe("ValidateImports", func() {
 			Expect(errorList).To(HaveLen(3))
 		})
 
-		It("should fail: missing API server config", func() {
-			landscaperImport.GardenerAPIServer = imports.GardenerAPIServer{
-				ComponentConfiguration: imports.APIServerComponentConfiguration{
-					CA:  &imports.CA{},
-					TLS: &imports.TLSServer{},
-				},
-			}
-
+		It("should fail: etcd Url has length 0", func() {
+			landscaperImport.EtcdUrl = ""
 			errorList := ValidateLandscaperImports(landscaperImport)
 			Expect(errorList).To(HaveLen(1))
 		})
+	})
 
-		It("should fail: etcd Url has length 0", func() {
-			landscaperImport.GardenerAPIServer = imports.GardenerAPIServer{
-				ComponentConfiguration: imports.APIServerComponentConfiguration{
-					Etcd: imports.APIServerEtcdConfiguration{
-						Url: "",
-					},
-					CA:  &imports.CA{},
-					TLS: &imports.TLSServer{},
-				},
-			}
+	Context("Etcd", func() {
+		It("should forbid invalid TLS configuration - etcd url is not set", func() {
+			landscaperImport.EtcdUrl = ""
 			errorList := ValidateLandscaperImports(landscaperImport)
-			Expect(errorList).To(HaveLen(1))
+			Expect(errorList).To(ConsistOf(
+				PointTo(MatchFields(IgnoreExtras, Fields{
+					"Type":  Equal(field.ErrorTypeInvalid),
+					"Field": Equal("etcdUrl"),
+				})),
+			))
+		})
+
+		It("should forbid invalid TLS configuration - etcd CA is invalid", func() {
+			landscaperImport.EtcdCABundle = pointer.String("invalid")
+			errorList := ValidateLandscaperImports(landscaperImport)
+			Expect(errorList).To(ConsistOf(
+				PointTo(MatchFields(IgnoreExtras, Fields{
+					"Type":  Equal(field.ErrorTypeInvalid),
+					"Field": Equal("etcdCaBundle"),
+				})),
+			))
+		})
+
+		It("should forbid invalid TLS configuration - etcd client certificate is invalid", func() {
+			landscaperImport.EtcdClientCert = pointer.String("invalid")
+			errorList := ValidateLandscaperImports(landscaperImport)
+			Expect(errorList).To(ConsistOf(
+				PointTo(MatchFields(IgnoreExtras, Fields{
+					"Type":  Equal(field.ErrorTypeInvalid),
+					"Field": Equal("etcdClientCert"),
+				})),
+			))
+		})
+
+		It("should forbid invalid TLS configuration - etcd client certificate is invalid", func() {
```

Comment:
The description of this IT is exactly the same with the one in L109.

```suggestion
		It("should forbid invalid TLS configuration - etcd client key is invalid", func() {
```